must-gather/gather

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+resources=()
+
+for I in $(/usr/bin/oc get crd | grep local.storage.openshift.io | awk '{print $1}'); do
+  resources+=($I)
+done
+
+for resource in ${resources[@]}; do
+  /usr/bin/oc get ${resource} --all-namespaces -o custom-columns=NAME:.metadata.name,NAMESPACE:.metadata.namespace --no-headers 2> /dev/null | \
+  while read ocresource; do
+    ocobject=$(echo $ocresource | awk '{print $1}')
+    ocns=$(echo $ocresource | awk '{print $2}')
+    if [ -z "${ocns}" ]|[ "${ocns}" == "<none>" ]; then
+      object_collection_path=must-gather/cluster-scoped-resources/${resource}
+      mkdir -p ${object_collection_path}
+      /usr/bin/oc get ${resource} -o yaml ${ocobject} > ${object_collection_path}/${ocobject}.yaml
+    else
+      object_collection_path=must-gather/namespaces/${ocns}/crs/${resource}
+      mkdir -p ${object_collection_path}
+      /usr/bin/oc get ${resource} -n ${ocns} -o yaml ${ocobject} > ${object_collection_path}/${ocobject}.yaml
+    fi
+  done
+done
+
+exit 0
```

Comment:
Can you add a quick doc about how to use must-gather with LSO? I would assume we will need more detailed docs in openshift-docs

Reply:
I've included docs in a separate commit, and referenced this from the main README. I've included the current image name, and will update it if this is modified.

Reply:
thank you. one more question - does this supplement image needs to be built and shipped by ART pipeline?

Reply:
It won't automatically get picked up. We'll need to file JIRAs to get ART to build it.